dev-requirements.txt

```diff
@@ -4,4 +4,5 @@ pytest-django
 pytest-cov
 django-debug-toolbar
 sphinx
-pre-commit
\ No newline at end of file
+pre-commit
+multiprocess
```

Comment:
this should go in the main requirements, not `dev`, since we need to run the index pages script in qa/prod